src/main/java/utils/Constants.java

```diff
@@ -6,6 +6,9 @@
 
 public class Constants {
     public static final String BASE_URL = "https://zenistock.zenitech.local/";
+    public static final String login = "https://zenistock.zenitech.local/login";
+    public static final String dashboard = "https://zenistock.zenitech.local/dashboard";
+    public static final String device_details = "https://zenistock.zenitech.local/device-details";
```

Comment:
we should have these uppercase since thats the structure we went for the constants class